apps/src/experiments.js

```diff
@@ -16,29 +21,61 @@ experiments.getQueryString_ = function () {
   return window.location.search;
 };
 
+experiments.getEnabledExperiments = function () {
+  var jsonList = localStorage.getItem(STORAGE_KEY);
+  var enabled = jsonList ? JSON.parse(jsonList) : [];
+  OLD_KEY_WHITELIST.forEach(function (key) {
+    if (localStorage.getItem('experiments-' + key) === 'true') {
+      enabled.push(key);
+    }
+  });
+  return enabled;
+};
+
+experiments.setEnabled = function (key, shouldEnable) {
+  var allEnabled = this.getEnabledExperiments();
+  if (!allEnabled.includes(key) && shouldEnable) {
+    allEnabled.push(key);
+  } else if (allEnabled.includes(key) && !shouldEnable) {
```

Comment:
In my latest commit I just switched to using indexOf. Sauce labs has temporarily disabled manual tests :( so this was faster than getting a proper ES5 test environment.